### DISCUSSION TITLE
How can I reused the loaded image?

### DISCUSSION THREAD

**Discussion by CyrilNb** [skydoves/landscapist#111] [Q&A] [answered]
Hello, 

I am using CoilImage to load from an URL my image. When I click on it I open a Details screen on a new activity. Is there a way to pass the already loaded image instead of loading it again in the new Activity?

Thanks for your answer!

**Comment by skydoves** [comment, marked as answer]
Hi @CyrilNb,
If you use the [caching strategy](https://coil-kt.github.io/coil/image_loaders/#caching) with your `ImageLoader`, the internal loader will reuse the cached bitmap automatically instead of requesting again from the provided URL.

**Reply by CyrilNb** [reply]
Alright, thanks for pointing this out. If I correctly understand, the caching strategy is already set by default (to 0.25%)? So I can increase this if needed.
